inc/profile.class.php

```diff
@@ -1450,7 +1453,10 @@ function showFormSetup($openform=true, $closeform=true) {
                             'field'     => 'notification'),
                       array('itemtype'  => 'SLA',
                             'label'     => __('SLA'),
-                            'field'     => 'sla'));
+                            'field'     => 'sla'),
+                     array('itemtype'  => 'LineOperator',
+                           'label'     => _n('Line operator', 'Line operators', Session::getPluralNumber()),
+                           'field'     => 'lineoperator'));
```

Comment:
I did not integrated lines with helpdesk because I thought it is better to associate the ticket with an asset. For example the phone using the line or the computer using the xDSL line.